gt7dashboard/gt7helper.py

```diff
@@ -766,3 +767,81 @@ def calculate_laps_left_on_fuel(current_lap, last_lap) -> float:
     laps_left: float
     fuel_consumed_last_lap = last_lap.fuel_at_start - last_lap.fuel_at_end
     laps_left = current_lap.fuel - (last_lap.laps_to_go * fuel_consumed_last_lap)
+
+
+
+def divide_laps(old_data: GTData, new_data: GTData):
```

Comment:
What does this do? I do not understand the whole logic behind `device_laps` and `equalizer_lap`. Please describe in the comment of the functions. Can you give a screenshot or something?